src/AbstractDatadirTestCase.php

```diff
@@ -16,19 +16,17 @@
 
 abstract class AbstractDatadirTestCase extends TestCase
 {
-    /** @var string */
```

Comment:
Nevim, jestli bych nedržel podporu pro 7.3... Čistě z toho důvodu, že jinak budeš na 7.4 muset updatnout všechno kde se to používá.

Reply:
pokud vydáme jako novou verzi, tak to updatovat nemusíme

Reply:
ale asi máš pravdu že s podporou 7.3 to bude asi lepší